gradle/compile-js-multiplatform.gradle

```diff
@@ -27,26 +31,30 @@ compileKotlinJs {
     kotlinOptions.metaInfo = true
     kotlinOptions.sourceMap = true
     kotlinOptions.moduleKind = 'umd'
-
-    kotlinOptions {
-        // drop -js suffix from outputFile
-        def baseName = project.name - "-js"
-        outputFile = new File(outputFile.parent, baseName + ".js")
```

Comment:
:warning: This could break compatibility with JS. Looking how to resolve this issue on plugin side.

Reply:
Resolved. Used manual test runner for classic JS backend.